### DISCUSSION TITLE
Pull request re-request after rebase

### DISCUSSION THREAD

**Discussion by ghsatpute-freshworks** [renovatebot/renovate#17459] [Request Help]
We've been using the Renovate bot for upgrading the dependencies. Sometimes after the PR is approved, Renovate rebases the PR. Due to our repository configuration, this makes the approvals invalid. Though renovate doesn't re-request the PR reviewers. 

Because of this, the PR is not visible on "Review Requested" section of github.com/pulls. Additionally, as this PR got stuck the next PR won't get raised by renovate as concurrency limit is set to k. 

I tried to check is there any way to configure Renovate to re-request PR reviews but I couldn't find any. What are my options here?

**Comment by rarkins** [comment]
Do you mean the list of configured reviewers which the bot requested when the PR was created? Or also any random user account which had approved already?

**Reply by ghsatpute-freshworks** [reply]
Re-review would be needed for all the users that had either been approved, rejected, or added a comment.

**Reply by rarkins** [reply]
Whether we do that would depend on whether it can be done elegantly or not. Requesting only the configured reviewers would likely be simpler, but maybe it doesn't matter

**Reply by ghsatpute-freshworks** [reply]
I think re-reviewing all would also work.

**Reply by ghsatpute-freshworks** [reply]
@rarkins  can we have this feature of request re-reviewing after PR is rebased?

**Reply by rarkins** [reply]
If you're volunteering to implement it, and it can be done elegantly, then a PR would be welcome
